lib/system/supports-colors.js

```diff
@@ -31,9 +31,8 @@ module.exports = (function () {
     return false;
   }
 
-  if (argv.indexOf('--color') !== -1 ||
-    argv.indexOf('--color=true') !== -1 ||
-    argv.indexOf('--color=always') !== -1) {
+  if (!argv.indexOf('--no-color') !== -1 ||
+    !argv.indexOf('--color=false') !== -1) {
```

Comment:
Hi this was just a copy of the above code except from I added ! so if --no-color is used color is disabled if it is not used then color is enabled by default.